lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -144,6 +168,32 @@ class AwsCompileEventBridgeEvents {
               );
             }
 
+            if (RetryPolicy) {
+              if (!shouldUseCloudFormation) {
```

Comment:
@evemontalvao Both `RetryPolicy` and `DeadLetterConfig` were only used when `shouldUseCloudFormation` is `true`. I'm not sure how to support those with the custom eventbridge resource, so I changed this to throw an error instead.

If we can easily add support for the custom resource, then we can change it.

Reply:
Well done cause actually we shouldn't support them with custom resources since it's being deprecated in the next version but I completely forgot to throw an error for that. Thanks